### ISSUE TITLE
Project Feedback!

### ISSUE THREAD

**Issue by codepathreview** [pm48/TwitterApplicationWithFragments#1]
@pm48: Please update your README to reflect the features you have implemented. Thank you.

**Comment by pm48** [comment]
Hi

I did not understand the above comment.
The readme is already updated:
https://github.com/pm48/TwitterApplicationWithFragments

Please let me know what I am missing.
I have put it based on the comments I received the previous week.

Thanks
Prerna

**Comment by codepathreview** [comment]
Noted on this, Prerna. There was just some confusion since the boxes were not ticked off.

**Comment by nesquena** [comment]
Yes, just so you know @pm48, Graders generally expect completed items to be marked off with `[x]` as explained here: https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments. Like a to-do list for easy tracking leading to the confusion

**Comment by pm48** [comment]
Thanks Nathan,understood.

**Comment by pm48** [comment]
Just for my knowledge,is there an easy way to mark these boxes in readme?
I usually open the README in vi and update it.
In vi I can just put a character x in those boxes.Is there a better way?

Thanks
Prerna

**Comment by nesquena** [comment]
Yes, that's basically all you need to do, is just mark an 'x' in the boxes and that's parsed as markdown into a checkbox once pushed to Github.

**Comment by codepathreview** [comment]
:+1: Nice work. A few notes after checking out the code:

* Nicely done architecting the fragments (base class and extension classes) such that you put as much code and logic as possible in the base class and then used abstract methods to define only the specific behavior where they differ
* Good to see you properly communicated the user to the user timeline fragment from the profile activity.
* Nice to see you used the ViewPager to properly setup the tabs
* Consider adding [progress bars](http://guides.codepath.com/android/Handling-ProgressBars#actionbar-progress-bar) to your app whenever a network request goes out
* Try to implement more optional features so that you will get to play with more widgets and understand complexities involved
* Considering this is 4th assignment and extension of last week's assignment, more efforts in UI polish would have been good.

Here's a detailed [Project 4 Feedback Guide here](http://courses.codepath.com/snippets/intro_to_android/project_4_feedback) which covers the most common issues with this submitted project. Read through the feedback guide point-by-point to determine how you might be able to improve your submission. 

This week (Week 5), we are going to cover the last major piece to the Android puzzle and that is using the hardware and SDK components such as the camera, photo gallery, location, maps, etc. After that, Week 6 and week 7 we will be covering a few important intermediate topics such as more about styling and animation as well as testing.

Following the bootcamp, we are going to have a public demo day to celebrate the progress you've all made with our next batch of Android students and multiple companies attending to see the group projects that you all have built. We are going to help however we can over the next few weeks to get the team project apps in shape for that.  
/cc @codepathreview

**Comment by pm48** [comment]
I had a question regarding my code in ProfileActivity:
I check for screenname and call the specific method accordingly.
showUser--which will display all tweets of the specific user
populateUserHeadline--which is for my profile.

Is there a better way of doing the same ?

if(screenName!=null) {
showUser(screenName);
}
else {
populateUserHeadline(user);
}
